lib/lz4.c

```diff
@@ -414,6 +414,7 @@ typedef enum { full = 0, partial = 1 } earlyEnd_directive;
 *  Local Utils
 **************************************/
 int LZ4_versionNumber (void) { return LZ4_VERSION_NUMBER; }
+const char* LZ4_versionString (void) { return LZ4_VERSION_STRING; }
```

Comment:
I know that `"hello" "world"` inside a `printf()` will work the same as `"hello world"`.
But I'm not sure if it also works outside of `printf()`, and in this specific case as a `return` argument.

I haven't tested it, so maybe just a simple test will answer my worries.